org.kicad.KiCad.ODBCDriver.sqliteodbc.yml

```diff
@@ -0,0 +1,49 @@
+id: org.kicad.KiCad.ODBCDriver.sqliteodbc
+branch: beta
+runtime: org.kicad.KiCad
+runtime-version: beta
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+appstream-compose: false
+build-options:
+  prefix: /app/extensions/ODBCDriver/sqliteodbc
+modules:
+  - name: sqliteodbc
+    cleanup:
+      - '*.a'
+      - '*.la'
+    config-opts:
+      - --with-odbc=/app
+    post-install:
+      # No idea why libdir is ignored, so clean up manually...
+      - install -d ${FLATPAK_DEST}/lib
+      - mv ${FLATPAK_DEST}/*.so ${FLATPAK_DEST}/lib/
+      - install -d ${FLATPAK_DEST}/template
+      - |
+        cat << EOF > ${FLATPAK_DEST}/template/sqliteodbc.ini
+        [SQLite3]
+        Description=SQLite3 ODBC Driver
+        Driver=/app/extensions/ODBCDriver/lib/libsqlite3odbc.so
+        Setup=/app/extensions/ODBCDriver/lib/libsqlite3odbc.so
+        Threading=2
+        EOF
+      - >
+        install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo \
+          org.kicad.KiCad.ODBCDriver.sqliteodbc.metainfo.xml
+      - >
+        appstream-compose \
+          --basename=org.kicad.KiCad.ODBCDriver.sqliteodbc \
+          --prefix=${FLATPAK_DEST} \
+          --origin=flatpak org.kicad.KiCad.ODBCDriver.sqliteodbc
+    sources:
+      - type: archive
+        strip-components: 2
+        url: http://www.ch-werner.de/sqliteodbc/sqliteodbc-0.9998.tar.gz
+        sha256: fabcbec73f98d1a34911636c02c29fc64147d27516b142e8e132c68c05a6065b
+        x-checker-data:
+          type: html
+          url: http://www.ch-werner.de/sqliteodbc/
+          version-pattern: <A HREF="sqliteodbc-([\d\.]+)\.tar\.gz">current source code</A>
+          url-template: http://www.ch-werner.de/sqliteodbc/sqliteodbc-$version.tar.gz
+      - type: file
```

Comment:
for the build file you need to add a source shell like this https://github.com/flathub/org.gimp.GIMP/blob/master/org.gimp.GIMP.json#L281